src/containers/Menu/Menu.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./Menu.scss";
+import { Link } from 'react-router-dom';
+
+import { UPCOMINGS } from '../../some-staff/consts';
+import { LIVE } from '../../some-staff/consts';
+import { LEAGUES } from '../../some-staff/consts';
+import { FAVORITE } from '../../some-staff/consts';
+
+const Menu = () => {
+  return (
+    <div className="main-menu" >
+      <ul>
+        <li><Link to={`/${UPCOMINGS}`} className={`nav_${UPCOMINGS}`} >{UPCOMINGS}</Link></li>
```

Comment:
```suggestion
        <li><Link to={AppRoutes.upcomings} className="nav-upcomings"} >{UPCOMINGS}</Link></li>
```
твои классы недолжны зависеть от названий роутов, потому что тогда изменение роута сломает стили.
и вообще зачем здесь классы?
и `_` обычно юзаются в BEM, только как `__` чтобы обозначить children блока, например
```css
.nav {}
.nav__li {}
```
вместо
```css
.nav {}
.nav > li {}
```